pip/commands/show.py

```diff
@@ -109,9 +109,9 @@ def search_packages_info(query):
         classifiers = []
         for line in metadata.splitlines():
             if not line:
-                break
+                pass  # The Problem SJB
```

Comment:
In what version of python can `splitlines` return a line as `None` ? :neutral_face: